src/Microsoft.Dnx.Tooling/Publish/PublishProject.cs

```diff
@@ -564,9 +569,7 @@ private bool GenerateWebConfigFileForWwwRootOut(PublishRoot root, Runtime.Projec
             var attributesToOverwrite = new[]
             {
                 "processPath",
-                "arguments",
-                "stdoutLogEnabled",
-                "stdoutLogFile"
```

Comment:
@muratg Those attributes should never be overwritten. It would break the configurations for people who know how to set them themselves and need automated deployments. The choice of log file path and server arguments should be left up to the user and not dnu since not everyone is using kestrel.